src/main/java/javax/measure/UnitConverter.java

```diff
@@ -53,35 +54,50 @@
 
     /**
      * Indicates if this converter is an identity converter. The identity converter returns its input argument ({@code convert(x) == x}).
+     * <p>
+     * Note: Identity converters are also always 'linear'. {@link UnitConverter#isLinear()}
```

Comment:
Was the lonely `{@link UnitConverter#isLinear()}` intended to be a `@see UnitConverter#isLinear()` (after `@return` tag)?

Reply:
Not sure, the intend is to provide more insight on the note itself (particularly the word 'linear') at the very same line.

Reply:
Then what about the following? (note the replacement of `@link` by `@linkplain`)

```
Note: Identity converters are also always {@linkplain UnitConverter#isLinear() 'linear'}.
```

Reply:
fine with me!

Reply:
... same thing also applies to a later note below

Reply:
I've updated this PR, but did not quite change the link as you suggest, but hope that suffices:
`Note: Identity converters are also always 'linear', see {@link UnitConverter#isLinear()}.`

Reply:
Cool, thanks.

Reply:
Could it be merged now?